7a_wqp_munge/src/munge_wqp_dat.R

```diff
@@ -67,28 +67,18 @@ munge_wqp_secchi <- function(outind, wqp_ind){
   gd_put(outind, outfile)
 }
 
-crosswalk_wqp_dat <- function(outind, wqp_munged, wqp_crosswalk, wqp_latlong_ind) {
+crosswalk_wqp_dat <- function(outind, wqp_munged, wqp_crosswalk) {
```

Comment:
not worth changing, but I have been trying to get in the habit of making these arg names align w/ the type. So when I see `wqp_munged`, I think it is an R object, vs `wqp_munged_ind` I would know is a file. Not a big deal at all, but something we might want to consider as we continue to evolve our collaborative pipelines - how much standardization do we do?

Reply:
I like that @jread-usgs -- I've found the standardization of organization and file names to be super helpful when developing, so I'm happy to implement that pattern.